website/docs/developer/development-guide/documentation/documentation-deployment.md

```diff
@@ -4,4 +4,73 @@ title: Deploying Documentation
 desc: Provides basic information on deploying ORCA documentation to nasa.github.io
 ---
 
-Information on deploying the documentation is TBD.
+The following sections provide information to ORCA users to create automated or
+manual releases on github. View the entire versioning and releases document 
+[here](https://github.com/nasa/cumulus-orca/blob/master/docs/release.md).
```

Comment:
We should add a card to suck that document `docs/release.md` into our website documentation somewhere Scott. No change needed for this PR just a a request.

Reply:
made a note to add a Jira card